.travis.yml

```diff
@@ -24,6 +23,10 @@ matrix:
       env:
         - LARAVEL_VERSION=^8.0
         - PHPUNIT_VERSION=^9.0
+    - php: "8.0snapshot"
```

Comment:
That's the only php 8 image Travis supports right now. Under `dist: trusty` there would be no php 8 image at all.